docs/source/running_bots.rst

```diff
@@ -0,0 +1,109 @@
+Running bots as participants
+============================
+
+Dallinger supports writing bots using the Selenium framework that participate in
+an experiment. Not all experiments have bots available, currently :doc:`demos/bartlett1932/index`
+and :doc:`demos/chatroom/index` have bots available.
+
+Writing a bot
+^^^^^^^^^^^^^
+
+In your ``experiment.py`` you will need to create a subclass of BotBase called Bot. 
+This class should implement the ``participate`` method, which will be called once
+the bot has navigated to the main experiment. Note, the BotBase class makes some
+assumptions about HTML structure, based on the demo experiments. If your HTML
+differs significantly you may need to override other methods too.
+
+
+.. currentmodule:: dallinger.bots
+
+.. autoclass:: BotBase
+  :members:
+
+Running bots locally
+^^^^^^^^^^^^^^^^^^^^
+
+You must set the configuration value ``recruiter='bots'`` to run an experiment using its
+bot. As usual, this can be set in local or global configurations, as an
+environment variable or as a keyword argument to :py:meth:`~dallinger.experiments.Experiment.run`.
+You should also set ``max_participants`` to the number of bots you want to run at once,
+``num_dynos_worker`` should be more than that number, as a bot takes up a worker
+processes. In addition, you may want to increase `num_dynos_web` to improve
+performance.
+
+Dallinger uses Selenium to run bots locally. By default, it will try to run
+phantomJS directly, however it supports using Firefox and Chrome through
+configuration variables.
+
+.. code-block:: ini
+
+    webdriver_type = firefox
+
+You can also provide a URL to a Selenium Webdriver hub, which is recommended if
+you're planning on running a large number of simultaneous bots. The hub does not
+need to be on the same computer as Dallinger, but it does need to be able to access
+the computer running Dallinger directly by its IP address.
+
+We recommend using FireFox when writing bots, as it allows you to visually see
+its output and allows you to attach the development console directly to the
+bot's browser session.
+
+Running an experiment with the API may look like:
+
+.. code-block:: python
+
+    participants = 4
+    data = experiment.run(
+        mode=u'debug',
+        recruiter=u'bots',
+        max_participants=participants,
+        num_dynos_web=int(participants/4),
```

Comment:
We're still on Python 2 for now, so this will set the number of web dynos to zero if the number of participants is < 4. How about `int(participants/4.0)`.